docusaurus/docs/reference/tunnelers/linux/linux-tunnel-troubleshooting.md

```diff
@@ -23,29 +23,69 @@ journalctl -xeu ziti-edge-tunnel.service
 
 ## Intercepting or hosting not working
 
-You may inspect the loaded identity and router info for a running `ziti-edge-tunnel` by dumping it to stdout or the systemd journal with an IPC command, or you may signal to dump the identities' info to a file.
+You may inspect the loaded identity and router info for a running `ziti-edge-tunnel` by dumping it to stdout or the systemd journal with an IPC command. Alternatively, you may signal to dump the identities' info into a file.
+
+```bash
+# dump identities info to std our journal if systemd unit with IPC command
+./ziti-edge-tunnel dump
+```
+
+```bash
+# dump identities info to a file and inspect
+sudo pkill -USR1 -f ziti-edge-tunnel
+less /tmp/ziti-dump.964315.dump
+```
 
-  ```bash
-  # dump identities info to std our journal if systemd unit with IPC command
-  ./ziti-edge-tunnel dump
-  ```
+Inspect the full command-line of the tunneler process
+
+```bash
+$ pgrep -f ziti-edge-tunnel | xargs ps -fww
+UID          PID    PPID  C STIME TTY      STAT   TIME CMD
+ziti      872509       1  0 10:42 ?        Ssl    1:54 /opt/openziti/bin/ziti-edge-tunnel run --verbose= --dns-ip-range=100.64.0.1/12 --identity-dir=/opt/openziti/etc/identities
+```
+
+Confirm the tunneler's tun interface is UP
+
+```bash
+$ ip link sh up tun0
+19: tun0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc fq_codel state 
+UNKNOWN mode DEFAULT group default qlen 500
+    link/none 
+```
 
-  ```bash
-  # dump identities info to a file and inspect
-  sudo pkill -USR1 -f ziti-edge-tunnel
-  less /tmp/ziti-dump.964315.dump
-  ```
+Find tunneler's nameserver IP
+
+```bash
+$ resolvectl --interface=tun0 dns
+Link 19 (tun0): 100.64.0.2
+```
+
+Query the Ziti nameserver
+
+```bash
+$ dig +noall +answer ssh.ziti.service.example.com @100.64.0.2
```

Comment:
another command without exposition. if you're trying to help someone, these all should be explained as to what they are doing and why. honestly, it feels like that sort of troubleshooting should be generified to any/all tunnelers and not just linux

Reply:
The troubleshooting steps will appear different, though they'll have the same motivations behind them across each OS/platform. It might be possible, but covering all those bases would be a more confusing document. The way this article is currently organized, it's Linux-specific, allowing it to remain focused, shorter, and more straightforward, which is appealing.

We've previously touched on the need for solving the "I need help" problem. I'm game to tackle that thought experiment with a generalized goal of helping everyone find the information they want.